### ISSUE TITLE
Some interfaces never triggered

### ISSUE THREAD

**Issue by Instein98** [microservices-demo/microservices-demo#859]
Hi, I am learning microservices through this project. I have successfully run this demo in docker. However, I noticed that for some interfaces in Carts, Orders, and Shipping, I can not trigger them by interacting with the front-end.

In detail, the following interfaces can not be triggered:
- ShippingController.getShipping
- ShippingController.getShippingById

Could you please tell me how to trigger those interfaces? Thanks!

**Comment by rodrmartinez** [comment]
Hi @Instein98 , how did you tried them? Did you registered a user and tried adding items to the cart and placing an order from the front-end?

**Comment by Instein98** [comment]
@rodrmartinez Yes. I have tried to operate from the front-end and the following interfaces seem never be triggered:
- ShippingController.getShipping
- ShippingController.getShippingById

Additionally, `CartsController.mergeCarts` can only be triggered by myself once.
Could you please explain what does `mergeCarts` mean and how to repeatedly trigger it? And for `getShipping` and `getShippingById`, are they deprecated?
